lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -3570,4 +3575,14 @@ public function getStringLiteralQuoteCharacter()
     {
         return "'";
     }
+
```

Comment:
I think the phpdoc is redundant

Reply:
Hi @simPod ,

Yeah, you are correct, it's totally useless and I will drop it soon.